src/microbe_stage/MembraneShapeGenerator.cs

```diff
@@ -303,7 +303,7 @@ private static ArrayMesh BuildEngulfMesh(Vector2[] vertices2D, int vertexCount,
             indices[index] = index;
             indices[index + 1] = index + 1;
 
-            vertices[index] = new Vector3(sourceVertex.X, height / 2, sourceVertex.Y);
+            vertices[index] = new Vector3(0.5f, height / 2, 0.5f);
```

Comment:
This still impacts the engulf mesh effect, in that this should be totally incorrect and should break the mesh shape, I don't see why this change should be a fine change. SOmeone will need to explain this to me for me to accept this.